composer/airflow_1_samples/requirements.txt

```diff
@@ -4,6 +4,7 @@ apache-airflow[gcp]==1.10.15
 apache-airflow-backport-providers-google==2021.3.3
 apache-airflow-backport-providers-cncf-kubernetes==2021.3.3
 kubernetes==11.0.0
+markupsafe==2.0.1
```

Comment:
Composer one's already been taken care of! No need to add it.

Reply:
Sounds good!